cl/aggregation/pool_impl.go

```diff
@@ -94,19 +95,40 @@ func (p *aggregationPoolImpl) AddAttestation(inAtt *solid.Attestation) error {
 	var mergedSig [96]byte
 	copy(mergedSig[:], merged)
 
-	// merge aggregation bits
-	mergedBits := solid.NewBitList(0, 2048)
-	aggBitsBytes := att.AggregationBits.Bytes()
-	inAttBitsBytes := inAtt.AggregationBits.Bytes()
-	for i := range aggBitsBytes {
-		mergedBits.Append(aggBitsBytes[i] | inAttBitsBytes[i])
-	}
-
-	// update attestation
-	p.aggregates[hashRoot] = &solid.Attestation{
-		AggregationBits: mergedBits,
-		Data:            att.Data,
-		Signature:       mergedSig,
+	epoch := p.ethClock.GetEpochAtSlot(att.Data.Slot)
+	clversion := p.ethClock.StateVersionByEpoch(epoch)
+	if clversion.BeforeOrEqual(clparams.DenebVersion) {
```

Comment:
why not clVersion <= clparams.DenebVersion?

Reply:
It's just my habit. I feel it would be clearer to compare versions using sentence-like statements rather than numerical operators, but I'm not very insistent on this.